desed_task/nnet/PSClassifier.py

```diff
@@ -0,0 +1,23 @@
+import torch.nn as nn
+import torch
+
+
+class PSClassifier(nn.Module):
```

Comment:
This would be the third module as described in Miyazaki et al.
According to their description, the architecture has three modules: a CNN-based feature extractor, Transformer or Conformer blocks, and a position-wise classifier. 
So this would be the position-wise classifier module. I kept all the modules separated because it sounds more organized to me and also because both Conformer and Transformer are using it so maybe it is useful to have the position-wise classifier in a module that can be imported. 
But we could discuss it and merge it somewhere else.